include/aspect/structured_data.h

```diff
@@ -132,6 +132,13 @@ namespace aspect
         load_file(const std::string &filename,
                   const MPI_Comm &communicator);
 
+        /**
+         * Fill the current object with data read from a NetCDF file
+         * with filename @p filename. This call will fail if ASPECT is not
+         * configured with NetCDF support.
+         */
+        void load_netcdf(const std::string &filename);
```

Comment:
Wouldn't you want to do the same here as with the previous function where you read only once and then distribute?

Reply:
Yes, but I was planning to do this in a separate PR.